### PR TITLE
fix: gmail email addresses dots gets removed

### PR DESCRIPTION
**Pull request by ANasouf**

### CODE REVIEW

File: packages/server/src/api/controllers/Contacts/Contacts.ts
Comment (abouolia) [inline packages/server/src/api/controllers/Contacts/Contacts.ts:1]: Your editor has different configuration, actually the problem not totally from editor, I think there's no `.prettierc` file under `/server` app like the one exist under `/webapp`, anyway let solve it temporary and remove the changes under this file out of PR.

Comment (ANasouf) [reply packages/server/src/api/controllers/Contacts/Contacts.ts:1]: Added a new commit, fixed formatting using vscode prettier extension using the same config used in webapp .prettierc

File: packages/server/src/api/controllers/Contacts/Contacts.ts [hunk 1]
```diff
@@ -153,7 +153,7 @@ export default class ContactsController extends BaseController {
       check('email')
         .optional({ nullable: true })
         .isString()
-        .normalizeEmail()
+        .normalizeEmail({ gmail_remove_dots: false })
```
Comment (abouolia) [inline packages/server/src/api/controllers/Contacts/Contacts.ts:156]: Let us remove the `normalizeEmail` function entirely, we want to give the user freedium to enter any email as long was email address.

**Review by abouolia — CHANGES_REQUESTED** [review]
Just one note.

File: packages/server/src/api/controllers/Contacts/Contacts.ts (hunk 1 above)
Comment (ANasouf) [reply packages/server/src/api/controllers/Contacts/Contacts.ts]: pushed new changes, i removed normalizeEmail function as requested.

**Review by abouolia — APPROVED** [review]

**Pull request merged**